src/components/Header.js

```diff
@@ -1,10 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({ changeUser, user, imgURL }) {
+  return (
+    <Wrapper>
+      <ProfileCard onClick={changeUser}>
+        <ProfileImage src={imgURL} width='70' height='70' />
+        <ProfileInfo>
+          <Name>{user}</Name>
+          <Status>현재 활동 중</Status>
+        </ProfileInfo>
+      </ProfileCard>
+    </Wrapper>
+  );
 }
-const Wrapper = styled.div`
-	background-color: black;
-	color: white;
+const Wrapper = styled.div``;
```

Comment:
헤더도 position:fixed 적용해주시면 좋을 것 같아요

```suggestion
const Wrapper = styled.div`
	position: fixed;
	width: 100%;`;
```